### DISCUSSION TITLE
Is there a sketch library for popovers/tooltips?

### DISCUSSION THREAD

**Discussion by travisterwilligar** [twilio-labs/paste#673] [Q&A] [answered]
(see title)

**Comment by serifluous** [comment, marked as answer]
Thanks, @travisterwilligar !

Popovers and tooltips aren't available in Sketch yet since we built them as alpha components.

We're in the process of adding a Tooltip sketch library, so if you need something to use right away, you can grab some layers from this file in this branch for now: https://share.goabstract.com/e3cace91-0504-4854-a03b-6abb29101146?sha=e51c8d4a9db9e265fdf3f6cdc7fac01ae059ad57

**Reply by travisterwilligar** [reply]
Thanks @serifluous!
